assets/js/main.js

```diff
@@ -1,172 +1,69 @@
-alert("Group 2 Project");
-// Every thiing under this point is used for the mobile responsiveness of the website 
-(function($) {
-
-	var	$window = $(window),
-		$body = $('body'),
-		$html = $('html');
 
-	// Breakpoints.
-		breakpoints({
-			large:   [ '981px',  '1680px' ],
-			medium:  [ '737px',  '980px'  ],
-			small:   [ '481px',  '736px'  ],
-			xsmall:  [ null,     '480px'  ]
-		});
 
-	// Play initial animations on page load.
-		$window.on('load', function() {
-			window.setTimeout(function() {
-				$body.removeClass('is-preload');
-			}, 100);
-		});
+(function(load){
+  var search = prompt("Type the name of a city ");
+  var url= "http://app.ticketmaster.com/discovery/v2/events.json?city="+search+"&apikey=IVeW1wnw1EgrDASBp2QqlmxszcLjjEKy";
 
-	// Touch mode.
-		if (browser.mobile) {
+    $.get(url).done(function(response){
+      console.log(response);
+      success(response);
+    });
 
-			var $wrapper;
+      function success(response){
 
-			// Create wrapper.
-				$body.wrapInner('<div id="wrapper" />');
-				$wrapper = $('#wrapper');
 
-				// Hack: iOS vh bug.
-					if (browser.os == 'ios')
-						$wrapper
-							.css('margin-top', -25)
-							.css('padding-bottom', 25);
+for (var i = 0; i < response._embedded.events.length; i++) {
 
-				// Pass scroll event to window.
-					$wrapper.on('scroll', function() {
-						$window.trigger('scroll');
-					});
+            // var newB = $("<button>");
+            var replace = $("<div><button>View Info</div>");
+            var newP = $("<p>");
+            
+            newP.text(response._embedded.events[i].name);
+            replace.append(newP);
+            // replace.append(newP,newB);
 
-			// Scrolly.
-				$window.on('load.hl_scrolly', function() {
 
-					$('.scrolly').scrolly({
-						speed: 1500,
-						parent: $wrapper,
-						pollOnce: true
-					});
 
-					$window.off('load.hl_scrolly');
+            $("#movies").append(replace);

```

Comment:
Is this ID "movies" intended to show the event? Why did you choose movie instead of event?